Runtime/Scripts/ObjImporter.cs

```diff
@@ -44,6 +44,9 @@ public class ObjImporter : MonoBehaviour
 
         public System.Action<GameObject> returnObjectTo;
 
+        public UnityEvent<bool> ObjImportSucceeded;
```

Comment:
gaan deze events in inspector gebruikt worden? zoniet zou ik misschien toch action gebruiken, maar dit is omdat ik dan kan zeuren :D

Reply:
Het kan in de inspector worden gebruikt ja